src/main/java/net/minestom/server/command/CommandManager.java

```diff
@@ -60,6 +60,18 @@ public synchronized void register(@NotNull Command command) {
         invalidateGraphCache();
     }
 
+    /**
+     * Register multiple {@link Command}.
```

Comment:
Should be plural "commands".

Reply:
Links to the actual Command class, would you prefer `Commands` or `{@link Command}s`?

Reply:
the second one